datamodel/referencedata.d/facilitytypes.csv

```diff
@@ -9,4 +9,5 @@ INTE,Inland terminal,"Inland terminal is a facility where containers are loaded,
 POTE,Port terminal,"Port terminal is a facility located adjacent to a waterway where containers are loaded, moved, or discharged onto/from sea-going vessels and barges. "
 PBPL,Pilot boarding place,"The place where a pilot boards the vessel upon arrival at the port boundaries."
 BRTH,Berth,"A designated location in a port or harbour used for mooring vessels when they are not at sea."
-RAMP,Ramp,"An inland container terminal (storing both full and empty containers) connected directly to a rail ramp where containers are loaded/discharged to/from a train."
\ No newline at end of file
+RAMP,Ramp,"An inland container terminal (storing both full and empty containers) connected directly to a rail ramp where containers are loaded/discharged to/from a train."
+ANCH,Anchorage,"An area in which vessels anchor or may anchor"
```

Comment:
```suggestion
ANCH,Anchorage,"The location in which vessels anchor or may anchor"
```

Reply:
I mentioned the "area" problem to Charles - he agreed to change it to location for now. We will later add a `geofence` location interface...